test/gas/gasLimit.js

```diff
@@ -0,0 +1,16 @@
+const assert = require("assert");
+const to = require("../../lib/utils/to.js");
+const preloadWeb3 = require("../helpers/web3/preloadWeb3");
+
+describe("options:gasLimit", () => {
+  const mnemonic = "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat";
+  const options = { mnemonic };
+  const services = preloadWeb3(options);
```

Comment:
why use this and not `getWeb3` - what's the difference?

Reply:
preloadWeb3 sets up a provider within a mocha `before` hook

Reply:
> preloadWeb3 sets up a provider within a mocha before hook.

If that's the only distinction, I think it's better to nix this function altogether. Better to make the `before` blocks explicit in the actual test file.

Reply:
Ok, can remove this.